server/routerlicious/packages/test-utils/src/testReadinessCheck.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Adding a duplicate of the IReadinessCheck interface from server-services-shared
+ * to prevent a cyclic dependency between server-services-shared and test-utils
```

Comment:
This would be resolved (I think) by moving these types to `services-core` which is a package specifically used for types.